src/v/serde/json/tests/BUILD

```diff
@@ -273,6 +273,7 @@ redpanda_cc_fuzz_test(
         "-rss_limit_mb=1024",
     ],
     data = glob(["testdata/fuzz/corpus/*"]) + ["testdata/fuzz/dict/json_fuzzer.dict"],
+    tags = ["manual"],
```

Comment:
shouldn't we still run the fuzz test in CI, not so that we get good fuzz coverage, but rather just so we know everything is compiling and working. so use like `-max_total_time=30` or whatever, in real fuzz testing we'd run for significantly longer.